tests/sentry/api/endpoints/test_source_map_debug.py

```diff
@@ -69,6 +69,115 @@ def test_no_frame_given(self):
         )
         assert resp.data["detail"] == "Query parameter 'frame_idx' is required"
 
+    @with_feature("organizations:fix-source-map-cta")
+    def test_frame_out_of_bounds(self):
+        event = self.store_event(
+            data={
```

Comment:
I'd create a const reference for this data since it looks like its the same for all three tests.